integration/boofcv-WebcamCapture/src/main/java/boofcv/io/gstwebcamcapture/OSHelper.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2021, Peter Abeles. All Rights Reserved.
+ *
+ * This file is part of BoofCV (http://boofcv.org).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package boofcv.io.gstwebcamcapture;
+
+/**
+ * Provides information on current OS
+ * @author Devin Willis
+ */
+public class OSHelper {
```

Comment:
I actually thought BoofCV had something similar at first, but it doesn't. We will eventually move it to another location so other modules can access it.